src/main/java/de/metas/ui/web/handlingunits/process/WEBUI_M_HU_MoveToAnotherWarehouse.java

```diff
@@ -46,76 +28,22 @@
  * #L%
  */
 
-/**
- * #2144
- * HU editor: Move selected HUs to another warehouse
- *
- * @author metas-dev <dev@metasfresh.com>
- *
- */
-public class WEBUI_M_HU_MoveToAnotherWarehouse extends HUEditorProcessTemplate implements IProcessPrecondition
+public class WEBUI_M_HU_MoveToAnotherWarehouse extends WEBUI_M_HU_MoveToAnotherWarehouse_Helper
```

Comment:
suggestion: rename this class to `WEBUI_M_HU_MoveToAnotherWarehouse_ExclQuarantined`
because that's the specific that it adds to `WEBUI_M_HU_MoveToAnotherWarehouse_Helper